js/components/App.js

```diff
@@ -1,13 +1,27 @@
 import 'babel/polyfill';
 
+// Mutations
+import ToggleWidgetEnableMutation from '../mutations/ToggleWidgetEnableMutation';
+
 class App extends React.Component {
+  _handleToggleEnabled(widget) {
+    Relay.Store.update(new ToggleWidgetEnableMutation({
+      widget: widget,
+    }));
+  }
+  
   render() {
     return (
       <div>
         <h1>Widget list</h1>
         <ul>
           {this.props.viewer.widgets.edges.map(edge =>
-            <li>{edge.node.name} (ID: {edge.node.id})</li>
+            <li>
+              {edge.node.name} (ID: {edge.node.id}) Enabled: {edge.node.enabled ? <span>TRUE</span> : <span>FALSE</span>}
+              <button onClick={this._handleToggleEnabled.bind(null, edge.node)} >
```

Comment:
Perhaps better practice to bind to `this`? Not necessary in this case because `handleToggleEnabled` doesn't actually use `this`, but the caller wouldn't necessarily know that.